tests/test_github.py

```diff
@@ -1741,3 +1741,50 @@ def test_github_api_pagination(self, tmp_path: Any) -> None:
         expected_status_output = '  develop *\n' + '\n'.join([f' |\n o-feature_{i}  rebase=no push=no'
                                                               for i in range(number_of_pages * prs_per_page)]) + '\n'
         assert_command(['status'], expected_status_output)
+
+    def test_github_enterprise_domain(self, mocker: Any) -> None:
+        git_api_state_for_test_github_enterprise_domain = MockGitHubAPIState(
```

Comment:
And this... is it needed? 🤔 given the GitHub domain mismatch

Reply:
Yeah, it's not